### PR TITLE
Remove pagination without results upload history

### LINKED ISSUE #5131: Hide pagination when no test results upload history
## Area or Component
Results > View upload history

## Quick Summary
There is an empty `<ol>` when there is no test upload history present.

### How to Replicate
Have an organization with no test results uploads. You will see an empty `nav` for pagination with an empty `<ol>`

## Action Requested
Do not include pagination where there is none.

## Source Code (optional)

[which file or line is the issue attached to?]: #

### PR DESCRIPTION
**Pull request by emyl3**
# FRONTEND PULL REQUEST

## Related Issue
Resolves #5131 

## Changes Proposed
- Hides pagination when there is no history of bulk test results upload
- Extends the border for "No results" to the entire width of the table (see screenshots below)

## Testing
- on dev2 for testing
- Check that pagination appears if you have at least 1 result
- Check that empty pagination is not there if you have no results
- Passes axe tests

## Screenshots / Demos
**Before**
Screen Shot 2023-02-10 at 14 32 23

**After**

No results
Screen Shot 2023-02-10 at 13 12 37

Results
Screen Shot 2023-02-21 at 11 44 30

### CODE REVIEW

**Comment by emyl3** [comment]
⚠️ I have no idea why the backend tests are failing after I rebased from `main`. Locally my `main` branch shows the same error that I see in CI: https://github.com/CDCgov/prime-simplereport/actions/runs/4148934178/jobs/7177647193 🤔 
Screen Shot 2023-02-10 at 19 26 30

**Comment by emyl3** [comment]
> ⚠️ I have no idea why the backend tests are failing after I rebased from `main`. Locally my `main` branch shows the same error that I see in CI: https://github.com/CDCgov/prime-simplereport/actions/runs/4148934178/jobs/7177647193 🤔 Screen Shot 2023-02-10 at 19 26 30

This is no longer an issue 😸

File: frontend/src/app/testResults/submissions/Submissions.test.tsx [hunk 1]
```diff
@@ -132,7 +132,7 @@ const getMocks = () => [
 ];
 
 describe("Submissions", () => {
-  it("should render no results", async () => {
+  it("should render no results and pagination", async () => {
```
Comment (johanna-skylight) [inline frontend/src/app/testResults/submissions/Submissions.test.tsx:135]: should this say "should render no results and no pagination"

**Review by johanna-skylight — CHANGES_REQUESTED** [review]
LGTM! I did find a mini typo but that was it 🎉

File: frontend/src/app/testResults/submissions/Submissions.test.tsx (hunk 1 above)
Comment (emyl3) [reply frontend/src/app/testResults/submissions/Submissions.test.tsx:135]: Thank you for your attention to detail!! 🤩 Will fix that up!

**Review by johanna-skylight — APPROVED** [review]
LGTM 🎉

**Review by BobanL — APPROVED** [review]
👍

**Pull request merged**